Packages/Newtonsoft.Json-for-Unity.Converters/Configuration/KeyedConfig.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Newtonsoft.Json.UnityConverters.Configuration
+{
+    [Serializable]
+    public struct KeyedConfig
```

Comment:
Codacy found an issue: [Implement 'IEquatable<T>' in value type 'KeyedConfig'.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)